tpm/aziot-tpm-rs/README.md

```diff
@@ -0,0 +1,4 @@
+# HSM Crate
```

Comment:
Doc is wrong.

Reply:
I went ahead and deleted the file. It should be pretty obvious that this is the Rust half corresponding to `aziot-tpm-sys`.